### DISCUSSION TITLE
Where can I find files : train.pkl and test.pkl

### DISCUSSION THREAD

**Discussion by house-chang** [logpai/loghub#34] [Q&A]
Dear Sir : 
I got error when run the command : 
$ python3 loglizer_demo.py
Traceback (most recent call last):
  File "loglizer_demo.py", line 58, in <module>
    with open(os.path.join(pkl_path, "train.pkl"), "rb") as fr:
FileNotFoundError: [Errno 2] No such file or directory: '../../proceeded_data/BGL/BGL_ws=60m_s=60m_0.8train/train.pkl'

where Where can I find files : train.pkl and test.pkl ? 
thank you
House

**Comment by zhujiem** [comment]
@JinYang88  Do you have these files?

**Reply by cxwshawn** [reply]
the loglizer project do not have train.pkl and test.pkl, where can i download them ?

**Reply by cxwshawn** [reply]
@JinYang88
